src/index.scss

```diff
@@ -0,0 +1,70 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  outline: 0;
+}
+
+:root {
+  font-size: 16px;
+}
+
+img {
+  max-width: 100%;
+}
+
+button,
+input,
+optgroup,
+select,
+textarea {
+  font-family: inherit; /* 1 */
+  font-size: 100%; /* 1 */
+  line-height: 1.15; /* 1 */
+  margin: 0; /* 2 */
+}
+
+body {
+  max-width: 1440px;
+  height: 100%;
+  width: 100%;
+  margin: 0 auto;
+  padding: 0;
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
+    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
+    sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  background-color: #f7f8fa;
+}
+
+a {
+  text-decoration: none;
+  color: #37464f;
+}
+
+code {
+  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
+    monospace;
+}
+
+.main-wrapper {
+  display: flex;
+  overflow: hidden;
+}
+
+.main-wrapper .page-wrapper {
+  min-height: 100vh;
+  background: #f7f8fa;
+  width: calc(100% - 280px);
```

Comment:
I am not comfortable with this calculation here. What happens on mobile where the device width is less than 320px? How many widths would be assigned?

Reply:
I didn't do this, I think is from the old design. I will change it to full width.